lib/dash.coffee

```diff
@@ -55,7 +57,17 @@ plugin = module.exports =
       path = activeEditor.getPath()
       language = activeEditor.getGrammar().name
 
-    spawn('open', ['-g', @createLink(string, path, language)])
+    if platform == "win32"
+      # Exec is used because spawn escapes arguments that contain double-quotes
+      # and replaces them with backslashes. This interferes with the ability to
+      # properly create the child process in windows, since windows will barf
+      # on an ampersand that is not contained in double-quotes.
+      quotedLink = '"' + @createLink(string, path, language) + '"'
+      exec('cmd.exe /c start "" ' + quotedLink, null)
```

Comment:
So in this case, the command that Windows needs in order to spawn the URI looks like this: 
```cmd.exe /c start "" "dash-plugin://keys=language&query=keyword"```

```start``` is basically the windows equivalent ```open``` but it only works from within the context of the ```cmd.exe``` interpreter kind of like ```/bin/bash``` commands.  So that explains the first part.   

If you tried to run ```start dash-plugin://keys=language&query=keyword``` on windows, it would only see ```dash-plugin://keys=language``` because the ```&``` is a reserved symbol inside of the ```cmd.exe``` interpreter.  To get around this with windows, you have to encapsulate the entire command to execute in double-quotes.  This would give you something that looks like this: ```start "dash-plugin://keys=language&query=keyword"```

But that doesn't work either, because ```start``` is particularly picky, if the first parameter passed to ```start``` is encapsulated in double-quotes, then ```start``` interprets it as the title for the window.  So, to work around that, we have to pass in the full command as ```start "" "dash-plugin://keys=language&query=keyword"```

You following me so far?  (Yes, this is pretty much ridiculous) 

So now we have the full command that needs to be used to dispatch the URI, we could try to pass this into ```spawn```.like so:

```spawn('cmd.exe', ['/c', 'start', '""', '"' + .... ])```

But for some cockamamie reason, node.js when running on win32 has been setup in such a way, that double-quotes are converted to backslashes when provided as parameters in ```spawn```. So the resulting command that windows sees when you attempt to execute the line above looks something like this:

```cmd.exe /c start \\ \dash-plugins://....\```

The above command is utter nonsense to Windows and won't run. In fact it will generate an alert in Windows complaining that the command could not be understood.  I was surprised when I was first testing and ran into these issues with ```spawn```.  It took digging into the node.js code to confirm that this is the expected behavior.   For this reason, ```exec``` was used instead, because it allows the entire command string to be passed to the OS without alteration.